host.json

```diff
@@ -2,12 +2,29 @@
   "version": "2.0",
   "logging": {
     "logLevel": {
-      "default": "Information"
+      "default": "Information",
+      "DurableTask.AzureStorage": "Warning",
+      "DurableTask.Core": "Warning",
+      "Function.GetLimitedProfile": "Warning",
+      "Function.GetLimitedProfileByPOST": "Warning",
+      "Function.GetMessage": "Warning"
```

Comment:
Why not even `GetService` and `GetUserServices`

Reply:
Simply I disabled information log level for GET functions with highest values in logs

<img width="1046" alt="Screenshot 2021-04-06 at 13 42 59" src="https://user-images.githubusercontent.com/59291437/113706049-813b8600-96de-11eb-9e4b-1e7feb0489b4.png">